src/modules/app/actions/listen-to-updates.js

```diff
@@ -68,6 +69,7 @@ export function listenToUpdates(history) {
             dispatch(updateAccountCancelsData({
               [log.marketID]: { [log.outcome]: [log] }
             }, log.marketID))
```

Comment:
The OrderCanceled event doesn't have a `marketID` field:
```
event OrderCanceled(address indexed universe, address indexed shareToken, address indexed sender, bytes32 orderId, Order.Types orderType, uint256 tokenRefund, uint256 sharesRefund);
```
If you need access to the market ID here, it needs to be looked up and attached to the log before it is re-emitted by augur-node (in src/blockchain/log-processors/order-canceled.ts).

Reply:
Note, I went ahead and submitted a PR to augur-node that adds marketID and outcome to the OrderCanceled event before it re-emits: https://github.com/AugurProject/augur-node/pull/187